tools/Ecoregionalization_workflow/cluster.xml

```diff
@@ -0,0 +1,132 @@
+<tool id="Clara_cluster" name="ClaraClust" version="0.1.0+galaxy0">
+    <description>Make cluster from BRT prediction with Clara algorithm</description>
+    <requirements>
+       <requirement type="package" version="4.2.3">r-base</requirement>
+       <requirement type="package" version="2.1.4">r-cluster</requirement>
+       <requirement type="package" version="1.1.1">r-dplyr</requirement>
+       <requirement type="package" version="2.0.0">r-tidyverse</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+         '$__tool_directory__/cluster_ceamarc.R'
+         '$input1'
+         '$input2'
+         '$input3'
+         '$k'
+         '$metric'
+         '$sample'
+         '$output1'
+         '$output2'
+         '$output3'
+    ]]></command>
+    <inputs>
+      <param name="input1" type="data" format="tsv" label="Prediction matrix (data to cluster from previous step) "/>
+      <param name="input2" type="data" format="txt,csv,tsv" label="Environment file"/>
+      <param name="input3" type="data" format="tsv" label="Prediction file (data.bio table from previous step)"/>
+      <param name="k" type="integer" label="Number of Cluster wanted" value= "2"/>
+      <param name="metric" type="select" label="What metric to use to calculate dissimilarities between observations ?">
+             <option value = "manhattan">manhattan</option>
+             <option value = "euclidean">euclidean</option>
+             <option value = "jaccard">jaccard</option>
+      </param>
+      <param name="sample" type="integer" label= "The number of samples to be drawn from the dataset" value="10"/>
```

Comment:
Is min/max here useful?

Reply:
Maybe min = "5" would be useful to prevent the user to enter a parameter which is not taken into account by the function used.

Reply:
top!